oda_api/secret.py

```diff
@@ -0,0 +1,11 @@
+import os
```

Comment:
```suggestion
from typing import Optional
import os
```

Reply:
@okolo , normally please have a look on this, but I will commit for now.